_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -5009,7 +5009,7 @@ mfxStatus MfxHwH264Encode::CheckMVCSeqDescQueryLike(mfxExtMVCSeqDesc * mvcSeqDes
         mvcSeqDesc->NumView = 0;
     }
 
-    if (mvcSeqDesc->NumOP > 0 && mvcSeqDesc->NumOP > 1024)
+    if (mvcSeqDesc->NumOP > 1024)
```

Comment:
This couple of lines above:
`if (mvcSeqDesc->NumView > 0 && (mvcSeqDesc->NumView > 2 || mvcSeqDesc->NumView < 2))`

Is it the same as:
`if (mvcSeqDesc->NumView > 0 && mvcSeqDesc->NumView != 2)`

Or I don't understand something?

Reply:
done